### DISCUSSION TITLE
WordPress module removal error

### DISCUSSION THREAD

**Discussion by wilfriedko** [ovh/manager#4403] [Q&A] [answered]
Hello, I cannot delete the wordpress module in my ovh hosting.
I am experiencing this error: An error occurred while requesting to delete the wordpress module. (User not granted for this request).

However, I have the administrator right.
I do not understand.

**Comment by antleblanc** [comment, marked as answer]
Hi @wilfriedko!

On my account I've deleted a WordPress module installed in a Hosting service and it has been deleted successfully.

Here is the API endpoint that must be called:
https://api.ovh.com/console/#/hosting/web/%7BserviceName%7D/module/%7Bid%7D#DELETE

If you are relying on the following URL (ca.ovh.com/manager), the right API endpoint is:
https://ca.api.ovh.com/console/#/hosting/web/%7BserviceName%7D/module/%7Bid%7D#DELETE

Here is the expected response (as an example):
```sh
200 DELETE /hosting/web/{serviceName}/module/{id}
{
  "doneDate": null,
  "id": 1111111111,
  "objectId": "1111111",
  "lastUpdate": "2021-02-09T14:12:03+01:00",
  "status": "todo",
  "startDate": "2021-02-09T14:12:03+01:00",
  "objectType": "Module",
  "function": "module/delete"
}
```

If you encounter an issue with your Hosting service, please create a support ticket.

Thank you!
